react/react1/week1/my-page/src/App.js

```diff
@@ -0,0 +1,20 @@
+import logo from "./logo.svg";
+import "./App.css";
+import { TodoList } from "./TodoList";
```

Comment:
Though it does the exact same thing, but I hope you know that if you want to import it as Default you could do it by `import TodoList from "./TodoList";` and mention export default on that component. As TodoList has only that.